conda_forge_tick/auto_tick.py

```diff
@@ -450,7 +450,6 @@ def migration_factory(
         migrator_config = loaded_yaml.get("__migrator", {})
         excluded_feedstocks = set(migrator_config.get("exclude", []))
         pr_limit = min(migrator_config.pop("pr_limit", pr_limit), pr_limit)
-        check_solvability = migrator_config.get('check_solvability', True)
```

Comment:
Why remove this? We want this for pypy

Reply:
We load the config in here: https://github.com/regro/cf-scripts/pull/961/files#diff-2be32f968602f542022009339eaac6afL422. The config dict will provide the `check_solvability` key for the pypy migration